scripts/main/mapview.js

```diff
@@ -281,6 +285,34 @@ mapview.open = function (albumID = null) {
 		updateZoom();
 	};
 
+	const setTrack = function (album) {
+		// add track
+		if (album.track_id) {
+			mapview.trackLayer = new L.GPX("uploads/tracks/" + album.track_id + ".xml", {
```

Comment:
If there was `album.track_short_path`, then it would not be necessary to hard-code any file path here. It would simply be `L.GPX(album.track_short_path)`.